pandas/tests/io/json/test_pandas.py

```diff
@@ -1296,25 +1296,6 @@ def test_datetime_tz(self):
         s_naive = Series(tz_naive)
         assert stz.to_json() == s_naive.to_json()
 
-    @pytest.mark.filterwarnings("ignore:Sparse:FutureWarning")
-    @pytest.mark.filterwarnings("ignore:DataFrame.to_sparse:FutureWarning")
-    @pytest.mark.filterwarnings("ignore:Series.to_sparse:FutureWarning")
-    def test_sparse(self):
-        # GH4377 df.to_json segfaults with non-ndarray blocks
-        df = pd.DataFrame(np.random.randn(10, 4))
-        df.loc[:8] = np.nan
-
-        sdf = df.to_sparse()
```

Comment:
I think you can actually do `sdf = df.astype('Sparse')` and for the rest keep this test? (so we test sparse series that way as well, not sure it is covered somewhere else)